test/DynamoCoreTests/Nodes/CustomDropdownTests.cs

```diff
@@ -40,12 +38,22 @@ public void OpenJsonDYNWithCorrectMenuItems()
             loader.LoadPackages(new List<Package> { pkg });
             // Assert expected package was loaded
             Assert.AreEqual(pkg.Name, "Dynamo Samples");
+        }
 
+        private void RunDropdownGraph()
+        {
             // Run the graph with correct info serialized, node should deserialize to correct selection
             string path = Path.Combine(TestDirectory, "pkgs", "Dynamo Samples", "extra", "CustomDropdownMenuNodeSample.dyn");
             RunModel(path);
+        }
 
-            Graph.Nodes.NodeModel node = CurrentDynamoModel.CurrentWorkspace.Nodes.FirstOrDefault();
+        [Test]
+        public void OpenJsonDYNWithCorrectMenuItems()
+        {
+            LoadSamplesPackage();
+            RunDropdownGraph();
+
+            NodeModel node = CurrentDynamoModel.CurrentWorkspace.Nodes.FirstOrDefault();
```

Comment:
Is there only one node in this graph?

Reply:
Two nodes. The first is the dropdown, the second is a watch node.